pkg/pillar/cmd/nodeagent/nodeagent.go

```diff
@@ -524,8 +531,14 @@ func handleLastRebootReason(ctx *nodeagentContext) {
 			if bootReason == types.BootReasonNone {
 				bootReason = types.BootReasonFirst
 			}
+		} else if previousSmartData.PowerCycleCount < smartData.PowerCycleCount {
+			reason = fmt.Sprintf("Reboot reason - power failure - at %s",
+				dateStr)
```

Comment:
String is slightly misleading because there was a power failure or manual power off at some time in the past, but dataStr is when EVE restarts.
Maybe saying "Reboot reason - device powered off. Restarted at %s"

Reply:
Also, to avoid inconsistencies, please move this into the BootReasonNone case below.

Reply:
Done